src/ESPEasyEth.ino

```diff
@@ -0,0 +1,90 @@
+
+#ifdef HAS_ETHERNET
+  // #define ETH_CLK_MODE ETH_CLOCK_GPIO17_OUT
+  // #define ETH_PHY_POWER 12
+  #include "ETH.h"
+  #include <esp_eth.h>
+
+String EthGetHostname()
+{
+  String hostnameToReturn(Settings.getHostname());
+  hostnameToReturn.replace(" ", "-");
+  hostnameToReturn.replace("_", "-"); // See RFC952
+  return hostnameToReturn;
+}
+
+bool checkSettings() {
```

Comment:
This is a rather generic function name.
Keep in mind this will be a global defined function.

Reply:
Fixed